### PR TITLE
chore: add minio deploy time bundle variable override definitions

### LINKED ISSUE #57: add minio deploy time bundle variable override definitions
### Describe what should be investigated or refactored

Add minio deploy time bundle overrides to the uds-k3d-dev package definition in the k3d-core-istio and k3d-core bundles

### Links to any relevant code

example of how this is handled for [istio](https://github.com/defenseunicorns/uds-core/blob/4a5a42c8fa9ee17656e462cb9df08562d9c85b96/bundles/k3d-standard/uds-bundle.yaml#L25) 

### Additional context

Deploy time helm variable overrides are the best option for someone configuring minio from the uds-k3d dev stack to be able to test their app. Overrides like what are already in place for istio from the uds-core package should be added to the uds-k3d-dev package to facilitate this customization.

### PR DESCRIPTION
**Pull request by zachariahmiller**
## Description

...

## Related Issue

Fixes #

Relates to #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md)(https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

### CODE REVIEW

File: bundles/k3d-istio/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+UDS Core groups foundational Unicorn Delivery Service applications that are heavily influenced [Big Bang](https://repo1.dso.mil/big-bang/bigbang).
+
+The core applications are:
+
+- [Istio](https://istio.io/) - Service Mesh
+
+The k3d uds-dev-stack provides:
+
+- [K3d](https://k3d.io/) - Containerized K3s Kubernetes Enviroment
```
Comment (mjnagel) [inline bundles/k3d-istio/README.md:14]: nit: should we include MetalLB here?

File: bundles/k3d-istio/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      uds-k3d-dev:
+        set:
+          buckets:
+            - name: "myfavoritebucket"
+                policy: "public"
+                purge: false
+          users:
+            - accessKey: console
+                secretKey: "console-secret"
+                policy: consoleAdmin
```
Comment (mjnagel) [inline bundles/k3d-istio/README.md]:
```suggestion
            - name: "myfavoritebucket"
              policy: "public"
              purge: false
          users:
            - accessKey: console
              secretKey: "console-secret"
              policy: consoleAdmin
```
spacing fixes

File: bundles/k3d-istio/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            - name: "myfavoritebucket"
+                policy: "public"
+                purge: false
+          users:
+            - accessKey: console
+                secretKey: "console-secret"
+                policy: consoleAdmin   
+```
+
+For more details on how to customize the Minio deployment, please see [Configuring Minio](https://raw.githubusercontent.com/defenseunicorns/uds-k3d/v0.2.1/docs/MINIO.md).
```
Comment (mjnagel) [inline bundles/k3d-istio/README.md]:
```suggestion
For more details on how to customize the Minio deployment, please see [Configuring Minio](https://github.com/defenseunicorns/uds-k3d/blob/main/docs/MINIO.md).
```
Rendered vs raw.

File: bundles/k3d-standard/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      uds-k3d-dev:
+        set:
+          buckets:
+            - name: "myfavoritebucket"
+                policy: "public"
+                purge: false
+          users:
+            - accessKey: console
+                secretKey: "console-secret"
+                policy: consoleAdmin
```
Comment (mjnagel) [inline bundles/k3d-standard/README.md]:
```suggestion
            - name: "myfavoritebucket"
              policy: "public"
              purge: false
          users:
            - accessKey: console
              secretKey: "console-secret"
              policy: consoleAdmin
```

File: bundles/k3d-standard/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+            - name: "myfavoritebucket"
+                policy: "public"
+                purge: false
+          users:
+            - accessKey: console
+                secretKey: "console-secret"
+                policy: consoleAdmin   
+```
+
+For more details on how to customize the Minio deployment, please see [Configuring Minio](docs/MINIO.md).
```
Comment (mjnagel) [inline bundles/k3d-standard/README.md]:
```suggestion
For more details on how to customize the Minio deployment, please see [Configuring Minio](https://github.com/defenseunicorns/uds-k3d/blob/main/docs/MINIO.md).
```

File: bundles/k3d-standard/README.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+- [Metrics Server](https://github.com/kubernetes-sigs/metrics-server) - Metrics
+- [Neuvector](https://open-docs.neuvector.com/) - Container Security
+- [Prometheus Stack](https://github.com/prometheus-operator/kube-prometheus) - Monitoring
+- [Promtail](https://grafana.com/docs/loki/latest/send-data/promtail/) - Log Aggregation
+- [Tempo](https://grafana.com/docs/tempo/latest/getting-started/) - Tracing
+- [Velero](https://velero.io/) - Backup & Restore
+
+The k3d uds-dev-stack provides:
+
+- [K3d](https://k3d.io/) - Containerized K3s Kubernetes Enviroment
```
Comment (mjnagel) [inline bundles/k3d-standard/README.md:26]: MetalLB here as well?

**Review by mjnagel — COMMENTED** [review]
With the update to latest uds-k3d we should be able to remove [this line](https://github.com/defenseunicorns/uds-core/blob/57-add-minio-deploy-time-bundle-variable-override-definitions/package.json#L21) since local path is in the dev stack namespace now.

Overall looks good, small comments on docs and change the title to a conventional commit syntax for CI.

File: bundles/k3d-istio/README.md (hunk 1 above)
Comment (zachariahmiller) [reply bundles/k3d-istio/README.md:14]: I'll add a note for metal lb

**Review by mjnagel — APPROVED** [review]
:shipit:

**Pull request merged**